secure-releases.md

```diff
@@ -0,0 +1,60 @@
+# Secure Releases and CVE Management for the JS Ecosystem
+
+## Secure Releases
+
+In order for your JS software project to be safe, secure, and reliable, the versions it releases need to follow security best practices. Below are the OpenJS Foundations recommended best practices for secure releases.
+
+### Two-factor Authentication
```

Comment:
consider expanding to MFA.  also if covering authentication, you may want to consider guidance on things like using a (good) password manager, best practices around storing recovery codes, etc.